server/proxy/pf_client.c

```diff
@@ -99,6 +99,8 @@ static BOOL pf_client_end_paint(rdpContext* context)
 static BOOL pf_client_pre_connect(freerdp* instance)
 {
 	rdpSettings* settings = instance->settings;
+	settings->DynamicResolutionUpdate = TRUE;
```

Comment:
will those settings be re negotiated later?

Reply:
no need for this, they are already negotiated in pf_common, will remove this, thanks